webdriver-spec.html

```diff
@@ -9401,6 +9401,7 @@ <h2>Acknowledgements</h2>
  <!-- David Burns --> <a href=http://www.theautomatedtester.co.uk/>David Burns</a>,
  <!-- Eran Messeri --> Eran Messeri,
  <!-- Erik Wilde --> Erik Wilde,
+ <!-- Geoffrey Sneddon --> Geoffrey Sneddon,
  <!-- Gábor Csárdi --> Gábor Csárdi,
```

Comment:
Actually, shouldn't my name be after Gábor? What locale are we using for the sorting? Under both English and Hungarian rules, I'd expect Geoffrey and Gábor.

Reply:
I think the problem here is that sort(1) uses `LC_COLLATE=C`, which sorts byte by byte.

I don’t understand why this doesn’t work though:

    % LC_ALL=en_GB.UTF-8 sort -
    Geoffrey Sneddon
    Gábor Csárdi^D
    Geoffrey Sneddon
    Gábor Csárdi

Reply:
I have manually fixed this.